aspnetcore/includes/managed-identities-test-non-production.md

```diff
@@ -0,0 +1,8 @@
+---
+author: wpickett
+ms.author: wpickett
+ms.date: 08/22/2024
+ms.topic: include
+---
+> [!NOTE]
+> This article uses a local database that doesn't require the user to be authenticated. Production apps should use the most secure authentication flow available. For more information on authentication for deployed test and production apps, see [Secure authentication flows](/aspnet/core/security/<!-- write create article something like secureAuthn-->).
```

Comment:
@wadepickett  create an issue to create `[Secure authentication flows](/aspnet/core/security/<!-- write create article something like secureAuthn-->).`

Reply:
Just to make sure I am in sync with the intent: 

- You added the commit that refers to a future topic not created so it is in the commit history along with Tom's last suggestion for it.
- Then you want me to pave over that with the last longer version of the include to merge today?

Reply:
Tracking issue: #33426 Secure Authentication Flows